.github/workflows/deploy.yml

```diff
@@ -20,8 +20,12 @@ jobs:
           python-version: '3.11'
       - name: Install the dependencies
         run: |
+          # install the JupyterLite dependencies
           python -m pip install jupyterlite-pyodide-kernel jupyterlite-core --pre
 
+          # install a couple of other useful packages for the demo
+          python -m pip install ipywidgets
```

Comment:
At some point we will probably want to move all these dependencies to a proper file.